src/pages/Lend/Filter/LendFilterPage.vue

```diff
@@ -318,6 +322,14 @@ export default {
 						opacity: 0.2;
 					}
 				}
+
+				.algolia-pagination-stats-component {
+					display: none;
+
+					@include breakpoint(1194px) {
+						display: block;
```

Comment:
I believe this needs to be namespaced into `filter-menu-pinned` to prevent it from displaying when the pinned experiment is inactive?

Reply:
We can merge this and make tweaks later as per Matt

Reply:
Will merge. Thanks.